yarn.lock

```diff
@@ -8442,16 +8442,17 @@ chrome-trace-event@^1.0.2:
   resolved "https://registry.yarnpkg.com/chrome-trace-event/-/chrome-trace-event-1.0.3.tgz#1015eced4741e15d06664a957dbbf50d041e26ac"
   integrity sha512-p3KULyQg4S7NIHixdwbGX+nFHkoBiA4YQmyWtjb8XngSKV124nJmRysgAeujbUVb15vh+RvFUfCPqU7rXk+hZg==
 
-chromedriver@92.0.0:
-  version "92.0.0"
-  resolved "https://registry.yarnpkg.com/chromedriver/-/chromedriver-92.0.0.tgz#7860efe7d082ec877dc8146aa920d30340fcbb47"
-  integrity sha512-IdJ5n5jLL6tCsGQF/fQHF2gDEhVzoYIqktUn6hE/BSsXlCcyDTi45fQbhTEhZlmshM8+BpnRtPuIT5DRbxNqKg==
+chromedriver@90.0.0:
```

Comment:
I think David just merged the 92 earlier today?

Reply:
Weird. I would've expected the CircleCI agents needing an update, but looks like the config hasn't changed... - https://github.com/justeat/fozzie-components/blob/master/.circleci/config.yml#L259

Reply:
@kevinrodrigues - Yeah as I thought, package.json is still at 90 in master.

Not sure how yarn.lock got out of sync.

I'll create a PR shortly to update Chromedriver + Circle config  - https://github.com/justeat/fozzie-components/blob/master/package.json#L66